cmake/projects/bison/hunter.cmake

```diff
@@ -6,14 +6,21 @@ include(hunter_download)
 include(hunter_pick_scheme)
 
 hunter_add_version(
-    PACKAGE_NAME
-    bison
-    VERSION
-    "3.0.4"
-    URL
-    "https://ftp.gnu.org/gnu/bison/bison-3.0.4.tar.gz"
-    SHA1
-    ec1f2706a7cfedda06d29dc394b03e092a1e1b74
+  PACKAGE_NAME
+  bison
+  VERSION
+  "3.0.4"
+  URL "https://ftp.gnu.org/gnu/bison/bison-3.0.4.tar.gz"
+  SHA1 "ec1f2706a7cfedda06d29dc394b03e092a1e1b74"
+)
+
+hunter_add_version(
+  PACKAGE_NAME
+  bison
+  VERSION
+  "3.0.4-p0"
+  URL "https://github.com/isaachier/bison/files/1471873/bison-3.0.4.tar.gz"
```

Comment:
I've created repo in `hunter-packages`: https://github.com/hunter-packages/bison